system/lib/libc/musl/src/stdio/__stdio_seek.c

```diff
@@ -4,7 +4,13 @@ off_t __stdio_seek(FILE *f, off_t off, int whence)
 {
 	off_t ret;
 #ifdef SYS__llseek
-	if (syscall(SYS__llseek, f->fd, off>>32, off, &ret, whence)<0)
+	off_t off_hi =
+#ifndef __EMSCRIPTEN__
+		off >> 32;
```

Comment:
Perhaps this should be fixed for all targets then, not just emscripten? Something upstream might be interested in too?